screens/auth/validation.js

```diff
@@ -27,6 +27,22 @@ const validation = {
     // }
   },
 
+  code: {
+    // This verifies that it's not blank.
+    presence: {
+      message: '^Code cannot be blank.',
+    },
+    length: {
+      is: 6,
+      message: '^Must be six digits!',
```

Comment:
Also this isn't a huge deal but it would be nice to have a maxLength on this. AuthTextField right now is `maxLength={fieldType === 'Phone Number' ? 10 : null}` and unfortunately I'm pretty sure you can't nest ternary operators.